admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -434,8 +213,8 @@ dependencies at once:
 
 .. _linux-ubuntu-bionic-install-atom:
 
-Download AtoM
-=============
+Download
+========
 
```

Comment:
I don't really mind, but curious why you chose to remove "AtoM" from the section header here?